### DISCUSSION TITLE
Suspected parameter incorrectly passed in @vue/babel-preset-app ?

### DISCUSSION THREAD

**Discussion by hejun200613224** [vuejs/vue-cli#6423] [Q&A]
In the dependent package `@vue/babel-preset-app`, there is a parameter `polyfills` with default value
 `['es.array.iterator', 'es.promise', 'es.object.assign', 'es.promise.finally']`

According to the documentation it's a list of core-js polyfills to pre-include when using `useBuiltIns: 'usage'`

But actually according to the source code, it was filtered in [this line](https://github.com/vuejs/vue-cli/blob/3b851a8dfe17895e7af894a70d65c3a23159041d/packages/%40vue/babel-preset-app/index.js#L198) and then pass to `envOptions.exclude` of `@babel/preset-env` in [this code](https://github.com/vuejs/vue-cli/blob/3b851a8dfe17895e7af894a70d65c3a23159041d/packages/%40vue/babel-preset-app/index.js#L219). The excerpt is as follows.
```
  const envOptions = {
    bugfixes,
    corejs: useBuiltIns ? require('core-js/package.json').version : false,
    spec,
    loose,
    debug,
    modules,
    targets,
    useBuiltIns,
    ignoreBrowserslistConfig,
    configPath,
    include,
    exclude: polyfills.concat(exclude || []),
    shippedProposals,
    forceAllTransforms
  }
```
Does this code work in the correct process?
I noticed that if this parameter is configured(even the default values too), the polyfiils will eventually be excluded.

**Comment by fangbinwei** [comment]
These polyfills will be injected in entry files by this [babel plugin](https://github.com/vuejs/vue-cli/blob/e661a923751c2f49a24cb065b5dd5999169e86af/packages/%40vue/babel-preset-app/polyfillsPlugin.js#L29-L39). Exclude these polyfills so that it will be injected once.
